src/SFA.DAS.EmployerIncentives.Web/Infrastructure/Configuration/WebConfigurationOptions.cs

```diff
@@ -3,11 +3,15 @@
     public class WebConfigurationOptions
     {
         public const string EmployerIncentivesWebConfiguration = "EmployerIncentivesWeb";
-
         public virtual string RedisCacheConnectionString { get; set; }
         public virtual string CommitmentsBaseUrl { get; set; }
         public virtual string AccountsBaseUrl { get; set; }
         public virtual string AllowedHashstringCharacters { get; set; }
         public virtual string Hashstring { get; set; }
+        public virtual string CdnBaseUrl { get; set; }
```

Comment:
CdnBaseUrl  and ZenDeskHelpCentreUrl  aren't being used.

Reply:
No worries. I'll take them out for now.